minicoro.h

```diff
@@ -631,6 +631,9 @@ typedef struct _mco_ctxbuf {
 static void _mco_wrap_main(void) {
   __asm__ __volatile__ (
     "movq %r13, %rdi\n\t"
+#if defined(__APPLE__)
+    "sub $8, %rsp\n\t"
+#endif
```

Comment:
Offsetting here will probably make the dummy return address endup in the wrong place. The dummy return address assists debugging as the last function in the backtrace when you put a breakpoint inside a coroutine. The offset should be really on `_mco_makectx` to overcome this.